lib/ss_light.go

```diff
@@ -382,18 +386,18 @@ func (l *LightClient) Start(
 
 	_, err = io.Copy(dst, rsc)
 	if err != nil {
-		return "Failed writing to destination", err
+		return NewOut(500, "Failed writing to destination", err.Error(), nil)
 	}
-
+	downloadTime := startTime - time.Now().Unix()
```

Comment:
This should be time.Now() - startTime right?

Reply:
yes, you are right